src/webapp/js/components/modal-backup.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+import Alert from 'react-s-alert';
+import _ from 'lodash';
+import Modal from 'react-modal';
+
+
+import ChwTable from '../container/chw-table';
+import apiClient from '../utils/api-client';
+import MotsConfirmModal from './mots-confirm-modal';
+
+class HealthWorkers extends Component {
+  constructor(props) {
+    super(props);
+    this.state ={
+      file:null,
+      content: ''
+    };
+
+    this.onFormSubmit = this.onFormSubmit.bind(this)
+    this.onChange = this.onChange.bind(this)
+    this.fileUpload = this.fileUpload.bind(this)
+    this.hideConfirmModal = this.hideConfirmModal.bind(this)
+    this.onSubmitModal = this.onSubmitModal.bind(this)
+    this.getModalParent = this.getModalParent.bind(this)
+  }
+
+  onFormSubmit(e){
+    e.preventDefault();
+    this.fileUpload(this.state.file)
+  }
+
+  onChange(e) {
+    this.setState({ file: e.target.files[0] })
+  }
+
+  fileUpload(file){
+    this.setState({ showConfirmModal: true });
+    const url = '/api/chw/upload';
+    const formData = new FormData();
+    formData.append('file', file);
+    const config = {
+      headers: {
+        'content-type': 'multipart/form-data'
+      }
+    };
+    return apiClient.post(url, formData, config).then((response) => {
+      console.log(response);
+      const message = 'test';
+      _.map(response.data,
+          error => `row ${error.id}: ${console.log(error)}`);
+
+      let wtf = '';
```

Comment:
you should consider renaming this

Reply:
yup this file wasn't meant to be included in the changes at all..